dockerfiles/benchmarks/push_bench_result.py

```diff
@@ -16,75 +17,110 @@
+
+EXAMPLE:
+./push_bench_result.py --type common \
+                       --project substrate-api-sidecar \
+                       --name sidecar \
+                       --result 34444 \
+                       --unit ms \
+                       --prometheus-server https://pushgateway.parity-build.parity.io
+Metric 'parity_benchmark_common_result_ms{project="substrate-api-sidecar",benchmark="sidecar"} 34444' was successfully   sent
 """
 
 import sys
 import argparse
 import requests
 
+
 def get_arguments():
-    parser = argparse.ArgumentParser(description='Push data to Victoria Metrics.')
-    parser.add_argument('--type','-t',
-                        default='common',
-                        choices=['common', 'specific', 'test'],
-                        type=str,
-                        help='Type of benchmark: common|specific|test')
-    parser.add_argument('--project','-p',
-                        type=str,
-                        required=True,
-                        help='Benchmark project (usually name of the repo)')
-    parser.add_argument('--name','-n',
-                        type=str,
-                        required=True,
-                        help='Benchmark name')
-    parser.add_argument('--result','-r',
-                        type=str,
-                        required=True,
-                        help='Result of benchmark')
-    parser.add_argument('--labels','-l',
-                        type=str,
-                        help='Dictionary with additional labels')
-    parser.add_argument('--unit','-u',
-                        type=str,
-                        default='ns',
-                        help='Metric unit (ns,s,bytes,info...)')
-    parser.add_argument('--prometheus-server','-s',
-                        type=str,
-                        required=True,
-                        help='Prometheus server address http://prom.com')
+    parser = argparse.ArgumentParser(
+        description="Push data to Victoria Metrics."
+    )
+    parser.add_argument(
+        "--type",
+        "-t",
+        default="common",
+        choices=["common", "specific", "test"],
+        type=str,
+        help="Type of benchmark: common|specific|test",
+    )
+    parser.add_argument(
+        "--project",
+        "-p",
+        type=str,
+        required=True,
+        help="Benchmark project (usually name of the repo)",
+    )
+    parser.add_argument(
+        "--name", "-n", type=str, required=True, help="Benchmark name"
+    )
+    parser.add_argument(
+        "--result", "-r", type=str, required=True, help="Result of benchmark"
+    )
+    parser.add_argument(
+        "--labels", "-l", type=str, help="Dictionary with additional labels"
+    )
+    parser.add_argument(
+        "--unit",
+        "-u",
+        type=str,
+        default="ns",
+        help="Metric unit (ns,s,bytes,info...)",
+    )
+    parser.add_argument(
+        "--prometheus-server",
```

Comment:
```suggestion
        "--prometheus-pushgateway",
```
I would change this also, to make it less confusing